server/pyproject.toml

```
@@ -17,7 +17,7 @@ pandas = "^1.2.5"
 pydantic = "^1.9.1"
 numexpr = "^2.8.1"
 typing_extensions = "^4.2"
-geopandas = "^0.10.2"
+geopandas = ">=0.11.1,<1"
```

```suggestion
geopandas = "^0.11.1"
```
We should be flexible in our libs but strict in our apps